src/main/kotlin/blackjack/view/output/ConsoleOutputView.kt

```diff
@@ -34,12 +34,12 @@ class ConsoleOutputView : OutputView {
     }
 
     override fun printPlayerRecords(playerRecords: PlayerRecords) {
-        println("## 최종 승패 ")
+        println("## 최종 수익 ")
         playerRecords.forEach(this::printPlayerRecord)
     }
 
     private fun printPlayerRecord(playerRecord: PlayerRecord) {
-        println("${playerRecord.player.name} :  ${playerRecord.toDisplayString()}")
+        println("${playerRecord.player.name} :  ${playerRecord.earnMoneyString}")
```

Comment:
```suggestion
        println("${playerRecord.player.name} :  ${playerRecord.earnMoney}")
```
이렇게 작성하셔도 무방합니다! earnMoneyString을 지울 수 있겠네요.

Reply:
네.. 감사합니다.  그런데, 추후  currency format 적용 등 변경이 있을 수 있을것 같아서요.